ttnn/cpp/pybind11/operations/__init__.hpp

```diff
@@ -91,6 +92,9 @@ void py_module(py::module& module) {
     auto m_embedding = module.def_submodule("embedding", "embedding operations");
     embedding::py_module(m_embedding);
 
+    auto m_embedding_backward = module.def_submodule("embedding_backward", "embedding backward operations");
```

Comment:
why each op here is in own submodule? it does not make sense

Reply:
Embedding and embedding_bw are it's own ops right now. Should we move them somewhere else?

Reply:
Either way, we can revisit this later